htdocs/class/snoopy.php

```diff
@@ -1151,7 +1151,7 @@ function _prepare_post_body($formvars, $formfiles)
                 break;
 
             case "multipart/form-data":
-                $this->_mime_boundary = "Snoopy" . md5(uniqid(microtime()));
+                $this->_mime_boundary = "Snoopy" . md5(uniqid(microtime(), true));
```

Comment:
Off topic? 

I would prefer not to modify snoopy.php, a deprecated vendor library. 

I really don't see the need for extra entropy in this case either. This 'uniqid' is only unique inside the single email being built.